app/forms/convicted_drug_felony_details_form.rb

```diff
@@ -0,0 +1,22 @@
+class ConvictedDrugFelonyDetailsForm < Form
+  set_attributes_for :interview,
+                     :convicted_drug_felony_name,
+                     :completed_probation_or_parole,
+                     :completed_treatment_program,
+                     :taken_action_towards_rehabilitation,
+                     :complied_with_reentry
+
+  validates_presence_of :convicted_drug_felony_name, message: "Make sure to include their name."
+
+  def save
+    enum_keys = %i[
+      completed_probation_or_parole
+      completed_treatment_program
+      taken_action_towards_rehabilitation
+      complied_with_reentry
+    ]
+    enum_values = attributes_for(:interview).slice(*enum_keys)
+    enum_values.update(enum_values) { |_, val| val == "1" ? "yes" : "no" }
```

Comment:
Can you explain why this is needed? Shouldn't they already be "yes" or "no"?

Reply:
they're only "yes" checkboxes, so they're either be yes or unfilled. since the user will have intentionally not checked them if they get to this page, we want to record them as nos